plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/index.tsx

```diff
@@ -17,7 +17,7 @@ import { FieldComponent as FieldComponentType, fieldName } from "./Fields/types"
 
 export type Props = {
   field: FieldComponentType;
-  dataID: string;
+  dataID?: string;
```

Comment:
Any reason you made this optional? dataID should always be present since it is required for all datasets